driver-core/src/main/java/com/datastax/driver/core/Host.java

```diff
@@ -194,12 +194,26 @@ public InetSocketAddress getSocketAddress() {
   }
 
   /**
-   * Returns the node broadcast address (that is, the IP by which it should be contacted by other
-   * peers in the cluster), if known.
+   * @return the node broadcast address, if known. Otherwise {@code null}.
+   * @see #getBroadcastSocketAddress()
+   * @see <a
+   *     href="https://docs.datastax.com/en/cassandra/2.1/cassandra/configuration/configCassandra_yaml_r.html">The
+   *     cassandra.yaml configuration file</a>
+   */
+  public InetAddress getBroadcastAddress() {
```

Comment:
This method is now used only in tests. Are you keeping it for binary compatibility? In this case we might want to mark it `@Deprecated` just as you did for `getListenAddress()`.

Reply:
I originally marked these methods deprecated, but then I backed off when I realized for most users the `InetSocketAddress` version will return port 0.

Reply:
Makes sense but then for consistency maybe we should _not_ mark `getListenAddress()` as deprecated?

Reply:
D'oh!  Didn't realize that, I'll make them consistent, thanks.

Reply:
We should at least explain that it duplicates information already present in the other method.